components/authz-service/storage/postgres/rule.go

```diff
@@ -17,6 +18,23 @@ const (
 )
 
 func (p *pg) CreateRule(ctx context.Context, rule *storage.Rule) (*storage.Rule, error) {
+
+	projectsFilter, err := projectsListFromContext(ctx)
+	if err != nil {
+		return nil, err
+	}
+	found := len(projectsFilter) == 0 // empty means all projects allowed
```

Comment:
perhaps we could lose the comment if we rename this var something like `allProjectsAllowed` ?